### ISSUE TITLE
Package dependency list

### ISSUE THREAD

**Issue by miamlevy** [ndrplz/dreyeve#18]
Hello,
Is there a package dependency list available for this code, specifically the semseg part? I am currently using Pyton 3.6.12 and having some dependency and compatibility issues with tensorflow and keras.
Thank you!

**Comment by ndrplz** [comment]
Hi @miamlevy unfortunately these are not available. These would be more than 4 years old anyway, and given the current rate of change for DL frameworks... well reproducing the same environment wouldn't probably be straightforward in any case.

I would suggest to simply rely on some more recent segmentation method for that part (e.g. start [here](https://paperswithcode.com/task/semantic-segmentation)).

Another thing I would consider is to completely ignore the semseg signal with a small loss of accuracy on the overall system (but better runtime).

**Comment by HHUBRO** [comment]
which version of python, cuda, theano and keras should I use
